crates/ruff_python_formatter/src/expression/expr_tuple.rs

```diff
@@ -61,47 +59,29 @@ impl FormatNodeRule<ExprTuple> for FormatExprTuple {
         } = item;
 
         // Handle the edge cases of an empty tuple and a tuple with one element
-        let last = match &elts[..] {
-            [] => {
-                return write!(
-                    f,
-                    [
-                        // An empty tuple always needs parentheses, but does not have a comma
-                        &text("("),
-                        block_indent(&dangling_node_comments(item)),
-                        &text(")"),
-                    ]
-                );
-            }
-            [single] => {
-                return write!(
-                    f,
-                    [group(&format_args![
-                        // A single element tuple always needs parentheses and a trailing comma
-                        &text("("),
-                        soft_block_indent(&format_args![single.format(), &text(",")]),
-                        &text(")"),
-                    ])]
-                );
-            }
-            [.., last] => last,
-        };
-
-        let magic_trailing_comma = use_magic_trailing_comma(f, last.range());
-
-        if magic_trailing_comma {
-            // A magic trailing comma forces us to print in expanded mode since we have more than
-            // one element
-            write!(
+        if let [] = elts.as_slice() {
```

Comment:
nit: i'd go for `is_empty()` or keep the match statement

Reply:
Yeah `is_empty` is much better